src/vss_tools/vspec/vssexporters/vss2samm/vss2samm.py

```diff
@@ -181,14 +163,13 @@ def cli(
 
     log.info("Loading VSS Tree...\n")
 
-    tree, datatype_tree = get_trees(
+    tree, _ = get_trees(
         vspec, include_dirs, aborts, strict, extended_attributes, uuid, quantities, units, types, overlays, expand
     )
 
     # Get the VSS version from the vss_tree::VersionVSS
     vss_version = __get_version_vss(tree)
-
-    __setup_environment(output_namespace, vss_version, split_depth)
+    cfg.init(output_namespace, vss_version, split_depth)
```

Comment:
At this stage VSSConcepts, vss_helper and tt_helper are loaded and any fields which depends on the user input, which is to be added to the initialization of the Config at this point will have Nulls in their values, as mentioned in above comments.